en/docs/install-and-setup/setup/security/general-data-protection-regulation-gdpr-for-wso2-api-manager.md

```diff
@@ -1,6 +1,6 @@
 # General Data Protection Regulation (GDPR) for WSO2 API Manager
 
-The Forget-Me tool pre-packed with API-M 2.2.0 can be used to remove identities of an **external user** who is deleted according to the system administrator's request. This tool removes user identities stored in the database and also in log files in order to meet GDPR requirements. The following sections guide you through configuring and running this tool in WSO2 API Manager.
+The Forget-Me tool pre-packed with API-M 3.2.0 can be used to remove identities of an **external user** who is deleted according to the system administrator's request. This tool removes user identities stored in the database and also in log files in order to meet GDPR requirements. The following sections guide you through configuring and running this tool in WSO2 API Manager.
 
 -   [Changing the default configurations of the tool.](#GeneralDataProtectionRegulation(GDPR)forWSO2APIManager-Changingthedefaultconfigurationsofthetool.)
```

Comment:
Do these sections have to be done in order?

Reply:
The order specified in the doc is correct. We need to edit the configs and config locations before running the tool .